patterns/jolla-hw-adaptation-cancro.yaml

```diff
@@ -90,7 +90,7 @@ Requires:
 - jolla-settings-system-flashlight
 
 # For sdcard on Mi4
-- sd-utils
+#- sd-utils
```

Comment:
Isnt this needed anymore? I remember this was added specifically for Mi4.

Reply:
No, it was my mistake, still needed :)